### ISSUE TITLE
bb-check does not have a valid role attribute

### ISSUE THREAD

**Issue by Blackbaud-PaulCrowder** [blackbaud/skyux1#464]
### Expected behavior

The `bb-check` span should have a `role` attribute of `checkbox`.
### Actual behavior

The `bb-check` span has a `role` attribute of `input` which is an "abstract" role type and not valid in markup.  This causes the Accessibility audit from Chrome's [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) to fail with [`AX_TEXT_01`](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01).
### Steps to reproduce
- Install Accessibility Tools and run an audit on a page with a `bb-check` element.
- Note the violation
### Plunker

http://plnkr.co/edit/Ya1dhdPfmB1WwAkYhsz4?p=preview

**Comment by Blackbaud-PatrickOFriel** [comment]
And a `tabindex="-1"`

**Comment by Blackbaud-PaulCrowder** [comment]
Do you mean it has that tabindex attribute or it should have that tabindex attribute?

**Comment by Blackbaud-PatrickOFriel** [comment]
It should have that attribute and does not currently. This allows tabbing properly in all browsers. It works now because some browsers interpret role and assume tabindex="-1" but we should make it explicit.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.5.15](https://github.com/blackbaud/skyux/releases/tag/v1.5.15)
